internal/resource/cluster.go

```diff
@@ -187,5 +202,11 @@ func (r *clusterResource) Delete(ctx context.Context, req resource.DeleteRequest
 }
 
 func (r *clusterResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
```

Comment:
be sure this gets documented effectively, not sure how the docstrings are bubbled up